app/views/events/index.html.erb

```diff
@@ -0,0 +1,42 @@
+<div class="row">
+<aside class="col-md-12">
+<section class="user_info">
+
+<h3>Upcoming Events (<%= @upcoming_events.count %>)<span><%= link_to "Create new", new_event_path %></span></h3>
+<% if @upcoming_events.any? %>
+  <div class="events">
+      <% @upcoming_events.each do |event| %>
```

Comment:
Hi @JamesOkunlade  and @AdnanAfsari ,

As @heyapricot previously mentioned you can use a partial for this instead of using `each`. This makes your views cleaner. It's one thing to make a project work. But a good engineer knows how to make a project work efficiently.